Certificate revocation documentation (#2986)
kyma-project/kyma: docs/application-connector/docs/08-02-get-client-certificate.md

This guide shows you how to get the client certificate.

>**NOTE:** The client certificate is valid for 92 days. See [this](#tutorials-renew-the-client-certificate) tutorial to learn how to renew the client certificate. 

> You can also revoke the client certificate, making it unable to be renewed. See [this](#tutorials-revoke-the-certificate) tutorial to learn how to do this.

Reviewer: ```suggestion
You can also revoke the client certificate, which prevents it from being renewed. See [this](#tutorials-revoke-the-client-certificate) tutorial to learn how to do this.
```
